StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp7/SpacingRules/SA1015CSharp7UnitTests.cs

```diff
@@ -50,5 +50,33 @@ public void TestMethod()
             await this.VerifyCSharpDiagnosticAsync(fixedCode, EmptyDiagnosticResults, CancellationToken.None).ConfigureAwait(false);
             await this.VerifyCSharpFixAsync(testCode, fixedCode, cancellationToken: CancellationToken.None).ConfigureAwait(false);
         }
+
+        /// <summary>
+        /// Verifies that a pattern in a switch statement that ends with a generic type does not produce a diagnostic.
```

Comment:
:memo: I've been treating `WorkItem` as sufficient to describe the reason why a *specific* regression test was created, but if you want to keep adding descriptive comments I won't stop you. 😄